README.md

```diff
@@ -203,6 +204,7 @@ Asynchronous concurrent distributed networking framework for PHP
 ### HTTP
 
 * [amphp/artax](https://github.com/amphp/artax) - An async HTTP/1.1 client built on [amp](#amphp)
+* [amphp/aerys](https://github.com/amphp/aerys) - A non-blocking HTTP application, WebSocket and file server
```

Comment:
Web Applications was admittedly used as a bit of a catch-all for WebSocket implementations, micro frameworks, and the like.

I'm fine with moving Aerys to the HTTP section if you think that's a better place for it. Just add a commit to this PR that removes it from Web Applications.

If you think Web Applications should be re-organized, I'm happy to consider that as well, though it should probably be in a separate PR.